reference/uitpas.json

```diff
@@ -2863,6 +2866,118 @@
         ],
         "description": "Retrieve information related to a chip number.\n\nIf the response contains `messages`, they MUST be displayed to the end-user.\n\nThe caller of this request must have `CHIP_NUMBERS_READ` permission."
       }
+    },
+    "/card-systems": {
+      "get": {
+        "summary": "Search card systems",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/CardSystemsPaginatedCollection"
+                },
+                "examples": {
+                  "Example": {
+                    "value": {
+                      "totalItems": 0,
+                      "member": [
+                        {
+                          "id": 0,
+                          "name": "string",
+                          "branding": {
+                            "logo": "string",
+                            "primaryColor": "string",
+                            "secondaryColor": "string"
+                          },
+                          "links": {
+                            "website": "string",
+                            "terms": "string"
+                          },
+                          "cities": [
+                            {
+                              "postalCode": "string",
+                              "name": "string"
+                            }
+                          ],
+                          "permanent": true
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request.Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/invalid-city\n* https://api.publiq.be/probs/uitpas/invalid-postal-code\n\nThe detail property might include more information for the client developer."
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          }
+        },
+        "operationId": "get-card-systems",
+        "description": "Search card systems.\n\nThis caller of this method, identified with client identification, does not require any permissions.",
+        "parameters": [
+          {
+            "schema": {
+              "type": "boolean",
+              "default": true
+            },
+            "in": "query",
+            "name": "permanentOnly",
+            "description": "Search only permanent card systems."
```

Comment:
If you use `permanentOnly=false`, will that then return only temporary card systems?